scripts/simtest/codecov.sh

```diff
@@ -11,6 +11,9 @@ fi
 # apply git patch
 git apply ./scripts/simtest/config-patch
 
+root_dir=$(git rev-parse --show-toplevel)
+export SIMTEST_STATIC_INIT_MOVE=$root_dir"/examples/move/basics"
```

Comment:
@halfprice and @mystenmark is this the correct `dir` to set? It did pass on my test run

Reply:
Yes, LGTM